Source/AppScaffolding/LibationScaffolding.cs

```diff
@@ -330,7 +330,18 @@ private static async System.Threading.Tasks.Task<(Octokit.Release, Octokit.Relea
 			//Download the release index
 			var bts = await gitHubClient.Repository.Content.GetRawContent(ownerAccount, repoName, ".releaseindex.json");
 			var releaseIndex = JObject.Parse(System.Text.Encoding.ASCII.GetString(bts));
-			var regexPattern = releaseIndex.Value<string>(ReleaseIdentifier.ToString());
+
+			string regexPattern;
+
+			try
+			{
+				regexPattern = releaseIndex.Value<string>(InteropFactory.Create().ReleaseIdentifier);
+			}
+			catch
+			{
+				regexPattern = releaseIndex.Value<string>(ReleaseIdentifier.ToString());
+			}
+
```

Comment:
Since the .deb and .rpm packages contain identical builds, there's no reason to try to differentiate between distributions in the ReleaseIdentifier. As far as Libation is concerned, Linux is Linux. The LinuxConfigApp will detect which package manager is installed. If it finds `yum` (the redhat package manager), then it appends "_RPM" to the release identifier.